src/AspNetCore.Authorization/GraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Server
+{
+    public static class GraphQLBuilderExtensions
+    {
+        /// <summary>
+        /// Add HTTP authorization services
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <returns></returns>
+        public static IGraphQLBuilder AddHttpAuthorization(this IGraphQLBuilder builder)
```

Comment:
Is the reason this is called `AddHttpAuthorization`, because subscriptions don't support auth?

Reply:
It was more to differentiate it from the authorization of the GraphQL nodes. 

I also added authorization support for the WebSocket transport. WebSockets still uses HTTP(S) before the connection is upgraded.